lib/bolt/cli.rb

```diff
@@ -818,71 +818,81 @@ def generate_types
     end
 
     # Initializes a specified directory as a Bolt project and installs any modules
-    # specified by the user, along with their dependencies
+    # specified by the user, along with their dependencies.
+    #
     def initialize_project
```

Comment:
I think it's probably time to move this into it's own class, similar to ProjectMigrate. I think it might make sense to combine those classes into a ProjectManager class that both initializes and migrates projects, but I don't think it'd be wrong per se to keep them separate, just more files.

Reply:
Updated!